app/src/main/java/com/sun/basic_japanese/data/source/KanjiDataSource.kt

```diff
@@ -0,0 +1,34 @@
+package com.sun.basic_japanese.data.source
+
+import com.sun.basic_japanese.data.model.*
+
+interface KanjiDataSource {
+
+    interface Local {
+        fun getKanjiBasicLocal(lesson: String, callback: OnDataLoadedCallback<KanjiBasicResponse>)
+
+        fun getKanjiAdvanceLocal(from: String, to: String, callback: OnDataLoadedCallback<KanjiAdvanceResponse>)
```

Comment:
Một dòng nên < 100 chars.
```suggestion
        fun getKanjiAdvanceLocal(
            from: String, to: String,
            callback: OnDataLoadedCallback<KanjiAdvanceResponse>
        )
```